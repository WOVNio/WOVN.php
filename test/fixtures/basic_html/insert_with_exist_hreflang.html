<!DOCTYPE html>
<html>
<head><meta charset="utf-8">
<link rel="alternate" hreflang="en" href="http://ja.localhost/en/t.php?hey=yo"><link rel="stylesheet" src="base.css"><link hreflang="vi" href="http://ja.localhost/vi/t.php?hey=yo" rel="alternate">
<link rel="stylesheet" src="fake.css">
</head>
<body>
<p>Fake body</p>
</body>
</html>
