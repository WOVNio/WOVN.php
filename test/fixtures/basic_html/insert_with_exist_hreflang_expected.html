<!DOCTYPE html>
<html>
<head><link rel="alternate" hreflang="en" href="http://my-site.com/en/"><link rel="alternate" hreflang="vi" href="http://my-site.com/vi/"><link rel="alternate" hreflang="zh-Hant" href="http://my-site.com/zh-CHT/"><link rel="alternate" hreflang="zh-Hans" href="http://my-site.com/zh-CHS/"><script src="//j.wovn.io/1" data-wovnio="key=123456&amp;backend=true&amp;currentLang=ja&amp;defaultLang=ja&amp;urlPattern=path&amp;langCodeAliases=[]&amp;langParamName=wovn" data-wovnio-info="version=WOVN.php_VERSION" async></script><meta charset="utf-8">
<link rel="stylesheet" src="base.css">
<link rel="stylesheet" src="fake.css">
</head>
<body>
<p>Fake body</p>
</body>
</html>
